app/src/gui/components/ui/survey-card-list.tsx

```diff
@@ -0,0 +1,36 @@
+import {ProjectInformation} from '@faims3/data-model';
+import SurveyCard from './survey-card';
+import {useNavigate} from 'react-router-dom';
+import * as ROUTES from '../../../constants/routes';
+
+/**
+ * SurveyCardList component that displays a list of SurveyCard components.
+ *
```

Comment:
By Survey here I think you mean Notebook/Survey - the thing who's name we configure.  Most other places in the code refer to this as Project so it might be better to use that term rather than Survey (or Notebook)